src/api/block_builder/pow/mod.rs

```diff
@@ -33,13 +53,12 @@ pub async fn finish_pow(client: &Client, payload: Option<Payload>) -> Result<Blo
         parent_blocks.dedup();
         let time_thread = std::thread::spawn(move || Ok(pow_timeout(tips_interval, cancel)));
         let pow_thread = std::thread::spawn(move || {
-            let mut client_miner = ClientMinerBuilder::new()
-                .with_local_pow(local_pow)
-                .with_cancel(cancel_2);
+            let mut client_miner = ClientMiner::builder().with_local_pow(local_pow).with_cancel(cancel_2);
```

Comment:
Thanks for the PR!

I think this needs to stay (import also needs to be updated)
```suggestion
            let mut client_miner = ClientMinerBuilder::new().with_local_pow(local_pow).with_cancel(cancel_2);
```
the `attempt to divide by zero` doesn't happen in Java alone and should be fixed with this change
Can you also redo your changes with the worker count for `promote_unchecked` and `reattach_unchecked`? They make sense

Reply:
Thanks! I didn't see that `ClientMinerBuilder::new` is different from the `Default` called by `::builder()`. 

Should the `Default` implementation be changed to always call `ClientMinerBuilder::new`?

```rust
fn new() -> Self {
    Self {
        worker_count: num_cpus::get(),
        ..Default::default()
    }
}
```

Reply:
Yeah that would make sense, the default now doesn't make a lot of sense
I would actually also change the `local_pow` to `true` as default/in the `new()` method